### ISSUE TITLE
GHC 8.0 

### ISSUE THREAD

**Issue by ocramz** [ghcformacosx/ghc-dot-app#49]
Are there any plans to ship a version that includes GHC 8 ?
Thanks

**Comment by etrepum** [comment]
I have a branch that builds 8.0.1 but I have had no time to test it so I have not done a release. I'm not currently using Haskell on a regular basis anymore. https://github.com/ghcformacosx/ghc-dot-app/tree/ghc-8.0.1

**Comment by ocramz** [comment]
I'll try the 8.0.1 branch, thank you; what type of testing did you perform on previous versions? In order to keep the project alive, would you be willing to release its ownership? I'd be happy to take care of it.

**Comment by etrepum** [comment]
Nothing formal. For both the current and oldest supported version of Mac OS X I would try the app to make sure that setup works, and then I'd build and use a few Haskell projects with it to make sure that it worked reasonably.

Would be happy to hand over ownership to anyone who wants to take care of it and has an active Apple Developer Program subscription to build the signed binaries.

**Comment by ocramz** [comment]
Ah, I hadn't thought of the need to sign binaries. I'm not an Apple developer, at least officially; I just happen to own a macbook and must cope with the idiosyncrasies of this not-quite-Unix system. However, I'd love to help out.

**Comment by etrepum** [comment]
Yeah, it's a $99/year expense that most Haskell users are probably not already paying.

The need for this is certainly a lot less than it used to be now that Stack exists, the only thing that this really buys you over just using Stack is that downloading it might be a little simpler.

**Comment by gbaz** [comment]
Note that the new platform for mac has a minimal version. It installs stuff into a frameworks folder in Library, etc, but other than that has the same sort of "package" that ghc dot app has in terms of just providing the basic binaries (ghc, cabal-install, alex, happy, stack) and not any additional libs.

**Comment by ocramz** [comment]
@etrepum would you be willing to advertise the need for a new maintainer for this project, e.g. on reddit? I'm sure there would be quite a few in the community with an Apple developer account willing to take this on.
